### PR TITLE
DS3231 Add individual alarm triggered and latch release

### PR DESCRIPTION
**Pull request by gijsn**
Hi,
Thanks for the great library. I was missing two functionalities, so I added them. 
These functions add the ability to individually check alarm one and two if they triggered, and also release the latch individually
Let me know if you require any changes

Best,

Gijsn

### CODE REVIEW

File: src/RtcDS3231.h [hunk 1]
```diff
@@ -568,6 +568,33 @@ template<class T_WIRE_METHOD> class RtcDS3231
...
+    }
+    DS3231AlarmFlag LatchAlarmTwoFlags()
+    {
+        uint8_t sreg = getReg(DS3231_REG_STATUS);
+        uint8_t alarmFlags = (sreg & _BV(DS3231_A2F));
+        sreg &= ~_BV(DS3231_A2F); // clear alarm flag 2
+        setReg(DS3231_REG_STATUS, sreg);
+        return (DS3231AlarmFlag)alarmFlags;
+    }
+    bool GetAlarmOneTriggered()
```
Comment (Makuna) [inline src/RtcDS3231.h:588]: make sure to have a blank line between all the new methods you added

**Review by Makuna — CHANGES_REQUESTED** [review]
Just some minor changes

**Comment by gijsn** [comment]
Thanks for the quick feedback, I made the requested changes. 
-gijsn

File: src/RtcDS3231.h [hunk 2]
```diff
@@ -569,33 +569,36 @@ template<class T_WIRE_METHOD> class RtcDS3231
         return (DS3231AlarmFlag)alarmFlags;
     }
   
-    DS3231AlarmFlag LatchAlarmOneFlags()
+        DS3231AlarmFlag LatchAlarmOneFlag()
```
Comment (Makuna) [inline src/RtcDS3231.h:572]: You added an extra indentation here, please remove.

**Review by Makuna — CHANGES_REQUESTED** [review]
just the minor extra indentation needs to be fixed.

File: src/RtcDS3231.h (hunk 2 above)
Comment (gijsn) [reply src/RtcDS3231.h]: Totally missed this, thanks!

**Review by Makuna — APPROVED** [review]

**Pull request merged**